tools/oss-fuzz-scanner/function_inspector.py

```diff
@@ -43,8 +45,13 @@ def print_function_details(project_name, functions_to_analyse):
 if __name__ == "__main__":
     if len(sys.argv) != 3:
         print(
-            "Usage: python3 function_inspector.py project_name function_name")
+            "Usage: python3 function_inspector.py project_name function_name language")
         sys.exit(0)
     project_name = sys.argv[1]
     function_name = sys.argv[2]
-    print_function_details(project_name, [function_name])
+    language = None
+    try:
+        language = sys.argv[3]
+    except IndexError:
+        pass
```

Comment:
could we have `language='c-cpp'` here just to avoid passing `language=None` into `print_function_details`, which I assume will produce an error?